recipe/meta.yaml

```diff
@@ -23,31 +23,32 @@ source:
     patches: CMakeLists-Zm10-win.patch  # [win]
 
 build:
-  number: 0
+  number: 1
+  skip: true  # [py!=310]
 
 requirements:
   build:
     - python                                # [build_platform != target_platform]
     - cross-python_{{ target_platform }}    # [build_platform != target_platform]
     - {{ compiler('cxx') }}
-    - {{ stdlib("c") }}
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
     - cmake
     - ninja
-    - {{ cdt('mesa-libgl-devel') }}  # [linux]
-    - {{ cdt('libxi-devel') }}       # [linux]
-    - {{ cdt('libxext-devel') }}     # [linux]
     - lld  # [win]
-    - libboost-devel 1.82.*
   host:
     - python
     - pybind11
     - pybind11-abi
     - occt {{ occt_ver }}
     - vtk
+    - libboost-devel
+    - libgl-devel   # [linux]
+    - xorg-libxi    # [linux]
+    - xorg-libxext  # [linux]
   run:
     - python
-    - {{ pin_compatible('occt', exact=True) }}
-    - {{ pin_compatible('vtk', max_pin='x.x') }}
+    - {{ pin_compatible('occt', max_pin='x.x.x') }}
```

Comment:
Pinning occt to exact will give you trouble when a new build number of occt comes out, our infrastructure won't retrigger your build.

Let vtk manage its own pinning with run exports.

Reply:
`ocp` is a bindings project for `occt`. The bindings code is typically tied to an exact version of `occt`, so we have to wait for the bindings to be (semi-manually) generated before we can release a new `ocp` conda package.

Would you still recommend against an exact pin in this case?